### PR TITLE
fixed typos + added 1 analyzer module for the calorimeter digis, 1 analyzer for the art Fragments and edited the already existing analyzer of the straw digis.

### PR DESCRIPTION
**Pull request by gianipez**
just a few, important, typos and syntax errors that needed to be fixed.

### CODE REVIEW

**Comment by gianipez** [comment]
Hi,

I integrated in this pull the following changes:
-> module to analyze the calorimeter digis
-> module to analyze the art::Fragments

I need these modules for validating the OTS workingflow. For the tracker, there was already a module.

Cheers,

Giani

**Comment by brownd1978** [comment]
@FNALbuild run build test

File: DAQ/src/StrawAndCaloDigisFromFragments_module.cc [hunk 1]
```diff
@@ -457,7 +458,7 @@ produce( Event & event )
 
   }
 
-  event.put(std::unique_ptr<EventNumber_t>(new EventNumber_t( eventNumber )));
+  //  event.put(std::unique_ptr<EventNumber_t>(new EventNumber_t( eventNumber )));
```
Comment (brownd1978) [inline DAQ/src/StrawAndCaloDigisFromFragments_module.cc:461]: Is there a reason this is commented and not removed?  Is it an optional data product?

File: Analyses/src/ReadStrawDigiReco_module.cc [hunk 2]
```diff
@@ -82,7 +82,7 @@ void mu2e::ReadStrawDigiReco::beginJob(){
 
   art::ServiceHandle<art::TFileService> tfs;
 
-  _hStrawId         = tfs->make<TH1F>( "StrawId",    "StrawId",                          180,   0.,   36000. );
+  _hStrawId         = tfs->make<TH1F>( "StrawId",    "StrawId",                          270,   0.,   54000. );
```
Comment (brownd1978) [inline Analyses/src/ReadStrawDigiReco_module.cc:85]: Can you explain what these numbers are in the comments?

**Comment by gianipez** [comment]
I removed the event number data-product because now we have an event header in the mu2eartdaq. In May, we will start a long discussion on that (Greg will coordinate it).
Eric suggested to remove it.

**Comment by kutschke** [comment]
This PR adds two new modules but the text summarizing the PR only mentions typos and syntax errors.    Please edit the text to give a more complete summary of the PR

**Comment by kutschke** [comment]
@FNALbuild run build tests

File: DAQ/src/StrawAndCaloDigisFromFragments_module.cc (hunk 1 above)
Comment (brownd1978) [reply DAQ/src/StrawAndCaloDigisFromFragments_module.cc:463]: My point was that you should delete the line not just comment it out, unless this is a future option, in which case it should be controlled but fcl.

**Comment by gianipez** [comment]
I removed the commented lines.

Thanks,

Giani

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Pull request merged**